www/docs/en/dev/guide/next/index.md

```diff
@@ -23,85 +23,62 @@ description: A look at topics that new Cordova developers will encounter.
 
 Even if you choose not to use Cordova, creating a mobile application without using a single page architecture will have serious performance implications. This is because navigating between pages will require scripts, assets, etc., to be reloaded. Even if these assets are cached, there will still be performance issues.
 
 Examples of SPA libraries you can use in your Cordova applications are:
 
-* [AngularJS](http://angularjs.org)
-* [EmberJS](http://emberjs.com)
-* [Backbone](http://backbonejs.org)
-* [Kendo UI](http://www.telerik.com/kendo-ui)
-* [Monaca](http://monaca.mobi/en/)
-* [ReactJS](http://facebook.github.io/react/)
-* [Sencha Touch](http://www.sencha.com/products/touch/)
-* [jQuery Mobile](http://jquerymobile.com)
+* [ReactJS](https://reactjs.org)
+* [Vue](https://vuejs.org/)
+* [Angular](https://angular.io/)
+* [EmberJS](https://emberjs.com)
+* [Backbone](https://backbonejs.org)
+* [Kendo UI](https://www.telerik.com/kendo-ui)
+* [Onsen UI](https://onsen.io)
+* [Sencha Ext JS](https://www.sencha.com/products/extjs/)
 
 And many, many, more.
 
 ## 2) Performance Considerations
 
 Consider the following issues to improve the performance in your mobile applications:
 
-**Click versus Touch** - The biggest and simplest mistake you can make is to use click events. While these "work" just fine on mobile, most devices impose a 300ms delay on them in order to distinguish between a touch and a touch "hold" event. Using `touchstart`, or `touchend`, will result in a dramatic improvement - 300ms doesn't sound like much, but it can result in jerky UI updates and behavior. You should also consider the fact that “touch” events are not supported on non-webkit browsers, see [CanIUse](http://caniuse.com/#search=touch). In order to deal with these limitations, you can checkout various libraries like HandJS and Fastclick.
+**Click versus Touch** - Many devices impose a 300ms delay on click events in order to distinguish between a click and double click event. This can have the effect of making your app feel slow and unresponsive. Avoiding this delay is one of the most important ways of improving your app's perceived performance. Most Android versions no longer impose this delay, but iOS by default still does. For iOS, using [WkWebView](https://github.com/apache/cordova-plugin-wkwebview-engine) instead of the default UIWebView gets rid of this delay. If you need to support older devices (Android 4.4 or iOS 8), consider a polyfill such as [FastClick](https://github.com/ftlabs/fastclick), or using `touchstart` and `touchend` instead of `click`.
```

Comment:
Maybe worth mentioning that this depends on the value in a viewport tag (in fact, maybe the viewport meta tag should be its own heading in here)

Reply:
I forgot about the need for the `meta` tag, I'll add a bit about that.